bokehjs/src/lib/models/widgets/tables/data_cube.ts

```diff
@@ -115,6 +113,27 @@ export class DataCubeProvider extends TableDataProvider {
     this.groupingInfos = groupingInfos
     this.toggledGroupsByLevel = groupingInfos.map(() => ({}))
 
+    const row_indices: (number[] | number)[] = this.target.get_array("row_indices")
+    const labels: string[] = this.target.get_array("labels")
```

Comment:
This is very nitpicky, but `get_array<Type>(...)` would be more idiomatic, meaning that you are querying arrays of the given type (albeit only on type level):
```suggestion
    const row_indices = this.target.get_array<number[] | number>("row_indices")
    const labels = this.target.get_array<string>("labels")
```